frontend/src/metabase/querying/components/FilterModal/CoordinateFilterEditor/CoordinateFilterEditor.tsx

```diff
@@ -85,23 +84,26 @@ export function CoordinateFilterEditor({
 }
 
 interface NumberValueInputProps {
-  column: Lib.ColumnMetadata;
   values: NumberValue[];
   valueCount: number;
   hasMultipleValues?: boolean;
   onChange: (values: NumberValue[]) => void;
 }
 
 function NumberValueInput({
-  column,
   values,
   valueCount,
   hasMultipleValues,
   onChange,
 }: NumberValueInputProps) {
   if (hasMultipleValues) {
     return (
-      <FilterValuePicker value={values} column={column} onChange={onChange} />
+      <FilterValuePicker
+        values={values.map(value => String(value))}
+        placeholder={t`Enter a number`}
+        getCreateLabel={query => (isFinite(Number(query)) ? query : null)}
+        onChange={values => onChange(values.map(value => Number(value)))}
```

Comment:
`Number("") === 0`, it's possible to enter whitespace and it will be converted to 0.
Previously spaces were not converted to zeros.

I think it'd be better to use `parseInt` and filter out `NaN`s (in `onChange`).

Reply:
Fixed